webapp/channels/src/components/user_settings/general/user_settings_general.tsx

```diff
@@ -393,6 +408,40 @@ export class UserSettingsGeneralTab extends PureComponent<Props, State> {
         this.submitUser(user, false);
     };
 
+    submitAttribute = async (settings: string[]) => {
+        const attributeID = settings[0];
+        const attributeValue = this.state.customAttributeValues[attributeID];
+        if (!this.state.customAttributeValues && !this.state.customAttributeValues[attributeID]) {
+            return;
+        }
```

Comment:
1/5: I guess I'm not seeing the need for `&& !this.state.customAttributeValues[attributeID]` as `!this.state.customAttributeValues &&` beforehand means that it would be false in order to match both conditions. Would `attributeValue == null` or `attributeValue === undefined` work here?

```suggestion
        const attributeValue = this.state.customAttributeValues?.[attributeID];
        if (attributeValue == null) {
            return;
        }
```

Reply:
Should have been an OR, but like your suggestion better.